man/man1/arcstat.1

```diff
@@ -451,5 +477,6 @@ Specify the sampling interval in seconds.
 
 .SH AUTHORS
 .LP
-arcstat was originally written by Neelakanth Nadgir and supported only ZFS ARC statistics.
+arcstat was originally written in Perl by Neelakanth Nadgir and supported only ZFS ARC statistics.
 Mike Harsch updated it to support L2ARC statistics.
+John Hixson ported it to Python for FreeNAS over some beer.
```

Comment:
Not sure if this is professional but I included it cause I thought it added a nice touch of humanity :P

Happy to get rid of it if anyone wants though

Reply:
I think it would be nice to acknowledge the other contributions as you did in the arcstat command itself by adding, "and has been improved by many people since."